README.md

```diff
@@ -6,8 +6,7 @@
 - ![Image build status for Manjaro](https://github.com/LukeShortCloud/winesapOS/actions/workflows/image-manjaro.yml/badge.svg)
 - ![Testing repository build status](https://github.com/LukeShortCloud/winesapOS/actions/workflows/repo-testing.yml/badge.svg)
 
-![winesapOS_Desktop_Screenshot 720p](https://github.com/LukeShortCloud/winesapOS/assets/10150374/10f43f3c-779a-4cf2-9485-603b2ece5b0f)
-
+![357951295-745c8b74-0137-4579-8342-758011ed6b07](https://github.com/user-attachments/assets/d66caf3f-7e82-4127-a689-c0e8b80dcc68)
```

Comment:
This needs to keep the text "winesapOS_Desktop_Screenshot 720p" (actually, we can remove the underscores). It is used by text readers for visually impaired folks to know what the image is supposed to represent.